mutmut/__main__.py

```diff
@@ -190,16 +190,23 @@ def exclude(context):
 
     mutations_by_file = {}
 
+    def add_mutations_by_file(mutations_by_file, filename, exclude, dict_synonyms):
+        mutations_by_file[filename] = count_mutations(
+            Context(
+                source=open(filename).read(),
+                filename=filename,
+                exclude=exclude,
+                dict_synonyms=dict_synonyms,
+            )
+        )
+
     for path in paths_to_mutate:
+        if isfile(path) and path.endswith('.py'):
+                filename = path
+                add_mutations_by_file(mutations_by_file, filename, exclude, dict_synonyms)
+                continue
```

Comment:
I'd prefer an else block here, I think that's a lot clearer than a continue and fall through.